src/components/Sidebar/Sidebar.tsx

```diff
@@ -62,6 +68,7 @@ const Sidebar = (): JSX.Element => {
           }}
         >
           <AddShoppingCartIcon />
+          <div className={style.number}>{qty !== 0 ? qty : ""}</div>
```

Comment:
it's more elegant avoid let and in this case it's easy to do with something like...

```
list.length ? list.length : 0
```

could be better to have it into the context ???

Reply:
It's not the same, because list.length is the quantity of differents products and qty is the total quantity of products in the shopping cart...
list.forEach((e) => {
    qty += e.qty;
  });

Reply:
ahhh good point, sorry I didn't realize of this, so I think we should use `reduce`, something like...

```
list.reduce((acc, current) => acc + (current.qty || 0), 0)
```

Reply:
even better??? although more inefficient on time with respect to the above.

```
list.map(item => item.qty).reduce((prev, next) => prev + next);
```